addons/xinference/Chart.yaml

```diff
@@ -21,4 +21,9 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
\ No newline at end of file
+appVersion: "1.16.0"
```

Comment:
shall change version to [0.8.0-alpha.5](https://github.com/apecloud/helm-charts/releases/tag/xinference-0.8.0-alpha.5)